workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentLauncher.cs

```diff
@@ -0,0 +1,334 @@
+                    return false;
+                }
+
+                if (string.IsNullOrEmpty(deploymentName))
+                {
+                    EditorUtility.DisplayDialog("Unable to start a deployment",
+                        "Please specify a valid deployment name.", "Close");
+                    return false;
+                }
+
+                var simSnapshotPath = GenerateTempSnapshot();
+
+                var arguments = new List<string>{
+                    "create",
+                    projectName,
+                    assemblyName,
+                    deploymentName,
+                    Path.Combine(ProjectRootPath, mainLaunchJson),
+                    Path.Combine(ProjectRootPath, snapshotPath)
+                };
+                if (simWorkerDeploymentEnabled)
+                {
+                    arguments.AddRange(new List<string>
+                    {
+                        deploymentName + "_sim_workers",
+                        simSnapshotPath,
+                        Path.Combine(ProjectRootPath, simWorkerLaunchJson)
+                    });
+                }
+
+                var processResult = await RedirectedProcess.RunInAsync(DotNetWorkingDirectory, Tools.Common.DotNetBinary, ConstructArguments(arguments), true, true);
+                return processResult.ExitCode != 0;
+            }
+
+            private async Task<bool> TriggerStopDeploymentAsync(string deploymentId)
+            {
+                if (!CheckDependencies())
+                {
+                    return false;
+                }
+
+                var arguments = new List<string>{
+                    "stop",
+                    projectName,
+                    deploymentId
+                };
+                var processResult = await RedirectedProcess.RunInAsync(DotNetWorkingDirectory, Tools.Common.DotNetBinary, ConstructArguments(arguments), false, true);
+                return processResult.ExitCode != 0;
+            }
+
+            private async Task<List<DeploymentInfo>> TriggerListDeploymentsAsync()
+            {
+                if (!CheckDependencies())
+                {
+                    return null;
+                }
+
+                var arguments = new List<string> {
+                    "list",
+                    projectName
+                };
+                var processResult = await RedirectedProcess.RunInAsync(DotNetWorkingDirectory, Tools.Common.DotNetBinary, ConstructArguments(arguments), false, true);
+
+                if (processResult.ExitCode != 0)
+                {
+                    return null;
+                }
+
+                // Get deployments from output.
+                var deploymentList = new List<DeploymentInfo>();
+                foreach (var line in processResult.Stdout)
+                {
+                    Debug.Log("Line: " + line);
+                    var tokens = line.Split(' ');
+                    if (tokens.Length != 2)
```

Comment:
Could you change the tool to output a specific marker at the beginning of an interesting line so this is a little less brittle?